src/services/formatters/YscopeFormatter/utils.ts

```diff
@@ -0,0 +1,189 @@
+import {Nullable} from "../../../typings/common";
+import {
+    COLON_REGEX,
+    DOUBLE_BACKSLASH_REGEX,
+    PERIOD_REGEX,
+    REPLACEMENT_CHARACTER,
+    REPLACEMENT_CHARACTER_REGEX,
+    SINGLE_BACKSLASH_REGEX,
+    YscopeFieldFormatterMap,
+    YscopeFieldPlaceholder,
+} from "../../../typings/formatters";
+import {JsonValue} from "../../../typings/js";
+import {LogEvent} from "../../../typings/logs";
+import {getNestedJsonValue} from "../../../utils/js";
+import RoundFormatter from "./FieldFormatters/RoundFormatter";
+import TimestampFormatter from "./FieldFormatters/TimestampFormatter";
+
+
+// Initialize commonly used regular expressions to facilitate reuse.
+const singleBackslashPattern = new RegExp(SINGLE_BACKSLASH_REGEX, "g");
+const doubleBacklashPattern = new RegExp(DOUBLE_BACKSLASH_REGEX, "g");
```

Comment:
I understand these are no longer exported; however, we still want to name all globals in `SCREAMING_SNAKE_CASE`.